model/qti/interaction/PortableCustomInteraction.php

```diff
@@ -232,7 +232,7 @@ public function feed(ParserFactory $parser, DOMElement $data, QtiNamespace $xmln
 
         $markupNodes = $parser->queryXPathChildren(['portableCustomInteraction', 'markup'], $data, $xmlnsName);
         if ($markupNodes->length) {
-            $markup = $parser->getBodyData($markupNodes->item(0), true, true);
+            $markup = $parser->getBodyData($markupNodes->item(0), true);
```

Comment:
I saw this true was added by Bertrand 7 years ago to fix pic/pic. Do you know if this wont bring this problem back?

https://github.com/oat-sa/extension-tao-itemqti/commit/7811fa6abd547c201b6267dcb1963fdcf4168555#diff-8f29b0ca7697b7fe1fcf6b3518e7f8c86076ef8b49b9afb3c12ff98e285ce254R159

Reply:
This is what I explained in the PR description. I did not find any reason for this. And for me, it creates more problems than it can solve.

The problem is that the XML validation does not support it after the browser has passed over the content. The browser accepts short notation for standalone tags, and automatically replaces them with the complete format (`<br>` is interpreted as `<br />`, then `</br>` does not exist, so it ignores it). This produces a broken XML in return. I could have added a filter to eliminate the dead tags before sending the data to the server, but it would add unnecessary complexity. I think it is better to remove a too greedy fix than add another layer of complexity which could introduce more issues. If the problem that was fixed 7 years ago arises again, I'm pretty sure we can fix it in a better way. But for now, I don't see what it could be, honestly.
Anyway, let me do some additional checks with various browsers. Maybe one is still not reformating the standalone tags to auto-closed ones, and in this case, it may also be an issue. But then we would need to apply a proper fix that is not toggling in one way or the other.

Reply:
Ok, lets go with this way and ask QA to validate the pic/pic case :)